contents/Documentation/Tutorials.playground/Pages/How to use nef library?.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,111 @@
+// nef:begin:header
+/*
+ layout: docs
+ */
+// nef:end
+
+// nef:begin:hidden
+import Bow
+Nef.Playground.needsIndefiniteExecution(false)
+// nef:end
+
+/*:
+ ## How to use `nef` library?
+ 
+ As you know, from Xcode 11, you can integrate package dependencies using SPM to share code between projects or even use third-party libraries. You can read more about it in Apple's article [Adding package dependencies to your app](https://developer.apple.com/documentation/xcode/adding_package_dependencies_to_your_app)
+
+ In `nef` [version 0.4](https://github.com/bow-swift/nef/releases/tag/0.4.0), we have modularized the core so that you can use nef as in CLI as in your macOS projects. Taking advantage of the integration of SPM in Xcode, you can easily import `nef` in your project.
```

Comment:
```suggestion
 Starting on `nef` [version 0.4](https://github.com/bow-swift/nef/releases/tag/0.4.0), we have modularized the core so that you can use nef as in library as in your macOS projects. Taking advantage of the integration of SPM in Xcode, you can easily import `nef` in your project.
```